sympy/series/sequences.py

```diff
@@ -711,6 +714,168 @@ def coeff_mul(self, coeff):
         return SeqFormula(formula, self.args[1])
 
 
+class RecursiveSeq(SeqBase):
+    """A finite degree recursive sequence.
+
+    That is, a sequence a(n) that depends on a fixed, finite number of its
+    previous values. The general form is
+
+        a(n) = f(a(n - 1), a(n - 2), ..., a(n - d))
+
+    for some fixed, positive integer d, where f is some function defined by a
+    SymPy expression.
+
+    Parameters
+    ==========
+
+    recurrence : SymPy expression defining recurrence
+        This is *not* an equality, only the expression that the nth term is
+        equal to. For example, if :code:`a(n) = f(a(n - 1), ..., a(n - d))`,
+        then the expression should be :code:`f(a(n - 1), ..., a(n - d))`.
+
+    y : function with argument
+        The name of the recursively defined sequence, e.g., :code:`y(n)`.
```

Comment:
This needs to be updated.

Reply:
Thanks for catching that; fixed in the latest commit.